MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/snapshot/SnapshotNotificationActivity.java

```diff
@@ -85,25 +85,29 @@ public boolean onMapClick(@NonNull LatLng point) {
    * @param width        of map
    */
   private void startSnapShot(LatLngBounds latLngBounds, int height, int width) {
-    if (!style.isFullyLoaded()) {
```

Comment:
Shall we remove the unused private field `style`?

Reply:
removed